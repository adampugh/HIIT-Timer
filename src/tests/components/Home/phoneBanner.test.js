import React from "react";
import { shallow } from "enzyme"; 

import PhoneBanner from "../../../components/Home/phoneBanner";

test("should render phoneBanner correctly", () => {
    const wrapper = shallow(<PhoneBanner />);
    expect(wrapper).toMatchSnapshot();
});

// test for modal opening function?
// handleOpenModal={false}