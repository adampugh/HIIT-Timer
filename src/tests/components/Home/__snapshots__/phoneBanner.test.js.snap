// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render phoneBanner correctly 1`] = `
<div
  className="phoneBanner"
>
  <div
    className="container"
  >
    <div
      className="phoneBanner__grid"
    >
      <img
        alt="phone"
        src="iphone.png"
      />
      <div>
        <h2>
          This is some text that is good but might not be. Don't ask me. Where Black Friday comsumers try to spend and where are those orders coming from.
        </h2>
        <h2>
          This is some text that is good but might not be. Don't ask me. Where Black Friday comsumers try to spend and where are those orders coming from.
        </h2>
        <button
          className="btn"
        >
          Sign Up
        </button>
      </div>
    </div>
  </div>
</div>
`;
